### PR TITLE
Add Etamin Studio

### PR DESCRIPTION
**Pull request by Spone**

### CODE REVIEW

**Comment by Spone** [comment]
Just a little detail: it was a bit unclear at first if `created_at` is meant to be the date the company was founded, or the date it was added to the `ruby-companies` repo, maybe the wording could be improved?

File: src/_companies/europe/france/auvergne-rhone-alpes/etaminstudio/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+stack:
+  - "rails"
+  - "postgres"
+  - "stimulus"
+
+created_at: 2023-03-20 09:47:00
+---
+
+We are passionate developers and designers, creators of quality web and mobile experiences.
```
Comment (CalvinWalzel) [inline src/_companies/europe/france/auvergne-rhone-alpes/etaminstudio/index.md:20]: Could you change this to be in the third person? I'd like to keep it consistent for all company descriptions :)

**Review by CalvinWalzel — COMMENTED** [review]
Hey @Spone, thanks for your contribution! I've added one comment, let me know what you think.

File: src/_companies/europe/france/auvergne-rhone-alpes/etaminstudio/index.md (hunk 1 above)
Comment (Spone) [reply src/_companies/europe/france/auvergne-rhone-alpes/etaminstudio/index.md:20]: Will do!

**Comment by CalvinWalzel** [comment]
> Just a little detail: it was a bit unclear at first if `created_at` is meant to be the date the company was founded, or the date it was added to the `ruby-companies` repo, maybe the wording could be improved?

Do you have a suggestion for a different name for the key? I've based this on the naming scheme for models in Rails and also added an explanation for the meaning in the readme, but I can see how this could be confusing.

**Comment by Spone** [comment]
> Do you have a suggestion for a different name for the key?

I updated the PR with a small tweak.

**Review by CalvinWalzel — APPROVED** [review]
Thanks, looks good!

**Pull request merged**